src/views/Nft/market/components/CollectibleCard/CardBody.tsx

```diff
@@ -1,21 +1,17 @@
 import React from 'react'
 import { Box, CardBody, Flex, Text } from '@pancakeswap/uikit'
-import minBy from 'lodash/minBy'
 import { useTranslation } from 'contexts/Localization'
 import { useBNBBusdPrice } from 'hooks/useBUSDPrice'
 import PreviewImage from './PreviewImage'
 import { CostLabel, MetaRow } from './styles'
 import LocationTag from './LocationTag'
 import { CollectibleCardProps } from './types'
 
-const CollectibleCardBody: React.FC<CollectibleCardProps> = ({ nft, nftLocation, currentAskPrice }) => {
+const CollectibleCardBody: React.FC<CollectibleCardProps> = ({ nft, nftLocation, currentAskPrice, lowestPrice }) => {
   const { t } = useTranslation()
-  const { name, image, tokens } = nft
+  const { name, image } = nft
   const bnbBusdPrice = useBNBBusdPrice()
 
-  const lowestPriceToken = tokens && minBy(Object.values(tokens), 'currentAskPrice')
```

Comment:
This actually haven't worked before. 
- In Newest component `tokens` were passed here as "hacked" tokens with only 1 element
- in CollectionNfts.tsx lowestPrice was already calculated and populated into `meta` property
- MoreFromThisCollection is skeleton UI component with bunch of BS inside

Lowest price comes as separate prop since it is easier to calculate it up in hierarchy.